api/registry/api/common.py

```diff
@@ -246,6 +253,9 @@ def get_score_history(
 If a specific address is provided, then the list will be of max length 1.\n
 \n
 Note: results will be sorted descending by `["created_at", "id"]`
+\n
```

Comment:
do we need this `history_endpoint` construct?
Why not just specify all this information in the api handler decorator itself

Reply:
I was thinking we would need it for specifying permissions specifically for this endpoint. It looks like this is how the rest of the api key permissions are being used

Reply:
Would be able to handle that in the ApiKey class? That is then used from both the Ninja API and the lambdas.

Reply:
✅